### PR TITLE
feat: add facility-language-server

### PR DESCRIPTION
**Pull request by bryankenote**
https://facilityapi.github.io/

Facilityapi is an open source interface definition language similar to protobuf. There are code generators for C#, Python, and Typescript. Currently they all implement http, but a facility service definition itself is protocol agnostic.

### CODE REVIEW

File: schemas/enums/language.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
         "Erg",
         "Erlang",
         "F#",
+        "FacilityServiceDefinition",
```
Comment (williamboman) [inline schemas/enums/language.json:54]: We can include spaces here, I think it'd be more correct to use:

```suggestion
        "Facility Service Definition",
```

File: packages/facility-language-server/package.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+source:
+  id: pkg:github/FacilityApi/FacilityLanguageServer@v2.1.0
+  asset:
+    - target: darwin_x64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: darwin_arm64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: win_x64
+      file: Facility.LanguageServer.zip:libexec/
```
Comment (williamboman) [inline packages/facility-language-server/package.yaml:22]: Should this not be `Facility.LanguageServer-win-x64.zip`? Also is there no Linux support?

Comment (ddunkin) [reply packages/facility-language-server/package.yaml:22]: `Facility.LanguageServer.zip` is platform neutral. `Facility.LanguageServer-win-x64.zip` contains an exe to make running on Windows easier for the Visual Studio extension.

File: packages/facility-language-server/package.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    - target: darwin_arm64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: linux_x64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: linux_arm64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: win_x64
+      file: Facility.LanguageServer.zip:libexec/
+    - target: win_arm64
+      file: Facility.LanguageServer.zip:libexec/
```
Comment (williamboman) [inline packages/facility-language-server/package.yaml:28]: Apologies for the back and forth on this one, but we can simplify this to a single platform-agnostic expression:

```suggestion
  asset:
    file: Facility.LanguageServer.zip:libexec/
```

**Pull request merged**